src/Mailer/Message/CitizenActionContactParticipantsMessage.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace AppBundle\Mailer\Message;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\EventRegistration;
+use Ramsey\Uuid\Uuid;
+
+final class CitizenActionContactParticipantsMessage extends Message
+{
+    /**
+     * @param EventRegistration[] $recipients
```

Comment:
We should keep only this line in the doc block no ?

Reply:
@B-Galati: if we specify a docblock, then it must be complete.